### PR TITLE
document competition

### PR DESCRIPTION
**Pull request by teto**
- doc: mention some of the competition

### CODE REVIEW

**Comment by NTBBloodbath** [comment]
Hey, seems like there are some merging conflicts. Mind cherry-picking these commits and making some small PRs for each? So we can merge them progressively and catch the conflicts (I really hate Git sometimes)

Cheers

**Comment by teto** [comment]
oups I based it off the wrong branch. It's now as I intented it to be sry ^^''

File: README.md [hunk 1]
```diff
@@ -157,6 +157,13 @@ request method (e.g. `GET`) and run `rest.nvim`.
 4. Push to the branch (<kbd>git push origin my-new-feature</kbd>)
 5. Create a new Pull Request
 
+
+## Related software
+
+- [Hurl](https://hurl.dev/)
+- https://httpie.io/
```
Comment (NTBBloodbath) [inline README.md:164]: Did you forget to use the markdown link syntax here or was it intentionally? :p

Comment (teto) [reply README.md:164]: intentional, how do you suggest I present it ?

**Review by NTBBloodbath — APPROVED** [review]

**Pull request merged**